### PR TITLE
Use modern convention for variable access

### PR DESCRIPTION
**Pull request by fbertsch**
There don't seem to be values for the env vars from the Airflow variables. Hoping this fixes that issue. (The env vars are present, but empty; other env vars (e.g. looker_hub_url) are present and valid.)

### CODE REVIEW

**Comment by fbertsch** [comment]
e.g. 
Screen Shot 2021-04-21 at 2 40 36 PM

**Comment by fbertsch** [comment]
Okay, this is useful. The build is failing because it says that variable is not available.

**Comment by jklukas** [comment]
> Okay, this is useful. The build is failing because it says that variable is not available.

In a recent PR, I had to modify `bin/run` to include a placeholder for the new variable I was using. The testing environment doesn't include the variables that are configured in the prod Airflow instance.

**Comment by fbertsch** [comment]
@jklukas yup, forgot we had added those awhile back. That should fix the dev build but doesn't fix the production issue.

File: bin/run [hunk 1]
```diff
@@ -139,6 +139,10 @@ init_variables() {
     airflow variables -s "taar_gcp_project_id" "taar_gcp_project_id"
     airflow variables -s "taar_dataflow_subnetwork" "taar_dataflow_subnetwork"
     airflow variables -s "taar_dataflow_service_account_email" "taar_dataflow_service_account_email"
+
+    airflow variable -s "looker_repos_secret_git_ssh_key_b64" "looker_api_client_id_staging"
```
Comment (jasonthomas) [inline bin/run:143]:
```suggestion
    airflow variable -s "looker_repos_secret_git_ssh_key_b64" "looker_repos_secret_git_ssh_key_b64"
```

File: bin/run [hunk 2]
```diff
@@ -139,6 +139,10 @@ init_variables() {
     airflow variables -s "taar_gcp_project_id" "taar_gcp_project_id"
     airflow variables -s "taar_dataflow_subnetwork" "taar_dataflow_subnetwork"
     airflow variables -s "taar_dataflow_service_account_email" "taar_dataflow_service_account_email"
+
+    airflow variable -s "looker_repos_secret_git_ssh_key_b64" "looker_repos_secret_git_ssh_key_b64"
```
Comment (jasonthomas) [inline bin/run:143]:
```suggestion
    airflow variables -s "looker_repos_secret_git_ssh_key_b64" "looker_repos_secret_git_ssh_key_b64"
```

File: bin/run [hunk 3]
```diff
@@ -139,6 +139,10 @@ init_variables() {
...
+    airflow variable -s "looker_api_client_id_staging" "looker_api_client_id_staging"
```
Comment (jasonthomas) [inline bin/run:144]:
```suggestion
    airflow variables -s "looker_api_client_id_staging" "looker_api_client_id_staging"
```

File: bin/run [hunk 4]
```diff
@@ -139,6 +139,10 @@ init_variables() {
...
+    airflow variable -s "looker_api_client_secret_staging" "looker_api_client_secret_staging"
```
Comment (jasonthomas) [inline bin/run:145]:
```suggestion
    airflow variables -s "looker_api_client_secret_staging" "looker_api_client_secret_staging"
```

**Review by haroldwoo — APPROVED** [review]

**Review by jasonthomas — APPROVED** [review]

**Pull request merged**